From ab2c04baf4afa52a22996bef6613682d6f965f85 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 14 Jul 2014 18:58:30 -0400
Subject: [idle] intel_idle: use CPU_TASKS_FROZEN instead of a numeric constant

Message-id: <1405364315-27440-3-git-send-email-prarit@redhat.com>
Patchwork-id: 85800
O-Subject: [RHEL7.1 PATCH BZ 1119374 2/7] intel_idle: use CPU_TASKS_FROZEN instead of a numeric constant
Bugzilla: 1119374
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Rik van Riel <riel@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1119374

commit e2401453111dd84da377e4e84740ab643885f95a
Author: Prarit Bhargava <prarit@redhat.com>
Date:   Wed Oct 23 09:44:51 2013 -0400

    intel_idle: use CPU_TASKS_FROZEN instead of a numeric constant

    Small fixup to use CPU_TASKS_FROZEN instead of 0xf.

    Signed-off-by: Prarit Bhargava <prarit@redhat.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/idle/intel_idle.c b/drivers/idle/intel_idle.c
index 7a0b537..df5a7e4 100644
--- a/drivers/idle/intel_idle.c
+++ b/drivers/idle/intel_idle.c
@@ -393,7 +393,7 @@ static int cpu_hotplug_notify(struct notifier_block *n,
  int hotcpu = (unsigned long)hcpu;
  struct cpuidle_device *dev;
 
- switch (action & 0xf) {
+ switch (action & ~CPU_TASKS_FROZEN) {
  case CPU_ONLINE:
 
   if (lapic_timer_reliable_states != LAPIC_TIMER_ALWAYS_RELIABLE)
-- 
1.7.1